src/main/assembly/dist/cfg/logback-service.xml

```diff
@@ -11,14 +11,29 @@
             <pattern>[%date{ISO8601}] %-5level %msg%n</pattern>
         </encoder>
     </appender>
+    <appender name="FILE-STATISTICS"
+              class="ch.qos.logback.core.rolling.RollingFileAppender">
```

Comment:
In http://logback.qos.ch/manual/appenders.html#TimeBasedRollingPolicy

> For various technical reasons, rollovers are not clock-driven but depend on the arrival of logging events. For example, on 8th of March 2002, assuming the fileNamePattern is set to yyyy-MM-dd (daily rollover), the arrival of the first event after midnight will trigger a rollover. If there are no logging events during, say 23 minutes and 47 seconds after midnight, then rollover will actually occur at 00:23'47 AM on March 9th and not at 0:00 AM. Thus, depending on the arrival rate of events, rollovers might be triggered with some latency. However, regardless of the delay, the rollover algorithm is known to be correct, in the sense that all logging events generated during a certain period will be output in the correct file delimiting that period.

Reply:
Removed the `file` property `<file>/var/opt/dans.knaw.nl/log/easy-download/easy-statistics.log</file>` from `FILE-STATISTICS` appender. This way the statistics log-file gets directly the date in its name, according to the `fileNamePattern` property `<fileNamePattern>/var/opt/dans.knaw.nl/log/easy-download/easy-statistics.%d{yyyy-MM-dd}.log</fileNamePattern>`.  This way we can ensure that the statisctics log-file with a date of the previous date exists when the script to send the log-files is executed.